web-ui/src/main/resources/catalog/locales/en-search.json

```diff
@@ -419,5 +419,52 @@
   "mdActions": "Available actions",
   "mdActions-view": "Viewable",
   "mdActions-download": "Downloadable",
-  "mdActions-process": "Processable"
+  "mdActions-process": "Processable",
+  "GUFrate": "Rating",
+  "GUFnoRatings": "No ratings <span class=\"fa fa-fw fa-star\">&nbsp;</span>",
+  "GUFfromRatingCountRatings": "from {{ratingCount}} ratings",
+  "GUFwaitingForApproval": "Waiting for approval",
+  "GUFseeAllFeedbacks": "See all feedbacks",
+  "GUFcommentsLastCommentOn": "{{userfeedbackCount}} comments, last comment on {{date}}.",
+  "GUFcountReviews": "{{userfeedbackCount}} Reviews",
+  "GUFdeleteUserFeedback": "Delete",
+  "GUFpublishUserFeedback": "Publish",
+  "GUFclose": "Close",
+  "GUFsave": "Save",
+  "GUFfeedbackOn": "Feedback on:",
+  "GUFnoComments": "No comments",
+  "GUFaddRating": "Review this dataset",
```

Comment:
Maybe we should make this more generic "Review this resource" because you may have a service, map, ...
![image](https://user-images.githubusercontent.com/1701393/38681980-4bf1c57e-3e6a-11e8-9143-7d56c67793c6.png)

or adapt the label depending on the hierarchylevel ?